content/projects/gsoc/2020/project-ideas/git-plugin-caching.adoc

```diff
@@ -0,0 +1,34 @@
+---
+layout: gsocprojectidea
+title: "Git repository caching on agents"
+goal: "Cache git repositories on agents for faster checkout"
+category: Plugins
+year: 2020
+status: draft
+sig: platform
+mentors:
+- "markewaite"
+- "fcojfernandez"
+- "rsandell"
+skills:
+- Java
+- Git
+links:
+  gitter: "jenkinsci/git-plugin"
+  draft: https://docs.google.com/document/d/1M9Na1eJxwGZ7fY1L-PsBGv62jL_pPBtqmGePpwogUBw
```

Comment:
There is a bug. When the status is `draft`, we will always use Google Doc instead of the Asciidoc text